### ISSUE TITLE
[Keyboard navigation-Help]Title property is not defined descriptive for Help control.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2538]
**User Experience:**
If any control does not have proper descriptive title then the person with visual related disability will find it difficult to interpret the proper fields information. This will hinder him to access all the controls of the page efficiently and also not compatible with AT.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Prerequisites:** Turn on Narrator.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section
4-Verify, If title property is defined descriptive for help control or not.

**Actual Result**
Title property is not defined descriptive for Help control. It is defined as References, lessons,.... 

**Expected Result**
Title property should be defined descriptive for Help control. 

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested fix:**

Using ARIA role to expose the control type of user interface component
e.g.
< div role="toolbar" tabindex="0" id="customToolbar" >
< img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home" >
 

Reference:
https://www.w3.org/TR/WCAG20-TECHS/ARIA16.html 
https://www.w3.org/TR/WCAG20-TECHS/ARIA4.html 
https://www.w3.org/TR/WCAG20-TECHS/ARIA5.html

Please Refer attachment for more details
![mas4 1 2_title is not enough descriptive for help](https://user-images.githubusercontent.com/30459830/28661474-6fceeeac-72d4-11e7-8236-50850b761e78.png)
[Help menu title is not descriptive.zip](https://github.com/Microsoft/pxt/files/1182511/Help.menu.title.is.not.descriptive.zip)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
